tools/mason/mason.chpl

```diff
@@ -1,3 +1,25 @@
+/*
+ * Copyright 2004-2017 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Author: Sam Partee
```

Comment:
FWIW, I don't think there's an established policy on including an author field in the top comment block like this (some might argue it's best left to the git history to keep track of). That said, I'm fine with it, and I see that @s-geronimoanderson included an author field in his work as well, so we might as well stay consistent for the Summer of 2017.